### DISCUSSION TITLE
What species (other than humans) play with snowballs? ☃️

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1277] [Galaxy Trivia] [answered]
**Bonus:** What would you do with a Monte Cristo?

**Comment by tomerf** [comment]
Japanese macaques, also known as snow monkeys, were the only other species documented playing with snowballs.

Bonus: A Monte Cristo is a sandwich, so obviously you would eat it 🤤

**Comment by peterbaccaro** [comment, marked as answer]
**Japanese Macaques**, also known as snow monkeys, are known for their love of playing in the snow. They have been known to make snowballs and play with them for fun. These snow monkeys have been observed rolling up snowballs, then pushing them down hills just for the heck of it.

A "Monte Cristo" is a type of sandwich made with ham, turkey and cheese which is then dipped in egg batter and fried.  It's often served with powdered sugar and jam. **Enjoying it as a meal would be a common choice.**

**Comment by peymansf2000** [comment]
Arctic foxes and Japanese macaques!
Bonus: A Monte Cristo is a sandwich, so obviously you would eat it 🤤
